tweb/src/main/frontend/src/js/vue/components/ProjectCard.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+	<div class="project-container">
+		<div class="project-icon">
+			<img v-bind:src="project.logoLocation">
+		</div>
+		<a v-bind:href="project.projectLink">
+			<div class="project-request-details" :style="{ 'background': project.overviewColor }">
+				<div class="project-request-name"><h3>{{ project.name }}</h3></div>
+				<div class="project-request-description"><p>{{ project.description }}</p></div>
+				<hr />
+				<div class="requestCountContainer">
+					<div class="requestNumber"><p>#{{ boundedRequests }}</p></div>
+					<div class="requestLabel"><p>active</p><p>requests</p></div>
+				</div>
+			</div>
+		</a>
+	</div>
+</template>
+
+<script lang="ts">
+    import {Component, Prop, Vue} from "vue-property-decorator";
+	import ProjectOverviewDetail from "../models/ProjectOverviewDetail"
+
+    @Component
+    export default class ProjectCard extends Vue {
+
+		defaultProject: ProjectOverviewDetail = {
```

Comment:
To make the object of type ProjectOverviewDetail, you can use 
`Object.assign(new ProjectOverviewDetail(), { name: "FundRequest", ....... });`
Else there can be errors on this (see ProjectsOverview).